src/tests/ilverify/TestDataLoader.cs

```diff
@@ -176,20 +176,37 @@ private static TheoryData<TestCase> GetTestMethodsFromDll(Func<string[], MethodD
                     var method = (EcmaMethod)testModule.GetMethod(methodHandle);
                     var methodName = method.Name;
 
-                    if (!String.IsNullOrEmpty(methodName) && methodName.Contains("_"))
+                    if (methodName?.Contains("_") == false)
+                        continue;
+
+                    var index = methodName.LastIndexOf("_Valid");
+                    if (index < 0)
+                        index = methodName.LastIndexOf("_Invalid");
+                    if (index < 0)
+                        continue;
+
```

Comment:
Is this change fixing any bugs?

Reply:
Yup, it was failing to perform transformation correctly when method name has a legit underscore. In this scope we need `methodName -> params` to undergo this transformation:

> special.set_MyProperty.set_MyProperty_Invalid_InitOnly' -> [ 'special.set_MyProperty.set_MyProperty', 'Invalid', 'InitOnly' ]